serverlessworkflow/modules/ROOT/pages/getting-started/java-embedded-workflows.adoc

```diff
@@ -78,11 +84,37 @@ Therefore, you can modify the previous example to generate the same output when
 <6> Build the workflow definition
 <7> Execute and print as in previous example
 
-
-== Further reading
+=== Additional fluent examples
 
 You can find additional and commented examples of fluent API usage (including jq expression evaluation and orchestration of rest services) link:{kogito_sw_examples_url}/sonataflow-fluent[here]
 
+== Dependencies explanation
+
+Embedded workflow uses a modular approach to keep the number of required dependencies as small as possible. The rationale is to avoid adding to the dependency set something that you are not going to use. For example, OpenAPI module is based on Swagger parser, if you are not going to call any OpenAPI service, it is better to avoid adding Swagger to the dependency set. This means the link:{swf_executor_core_maven_repo_url}[Core] dependency does not include the stuff to use gRPC, OpenAPI or most custom function types.  
```

Comment:
Just rephrased the sentence. 
```suggestion
Embedded workflow uses a modular approach to keep the number of required dependencies as small as possible. The rationale is to avoid adding something that you will not use to the dependency set. For example, the OpenAPI module is based on a Swagger parser, if you are not going to call any OpenAPI service, it is better to avoid adding Swagger to the dependency set. This means the link:{swf_executor_core_maven_repo_url}[Core] dependency does not include the stuff to use gRPC, OpenAPI, or most custom function types.  
```